lib/topological_inventory/orchestrator/worker.rb

```diff
@@ -362,6 +363,15 @@ def cleanup_errored_deployment_configs
         end
       end
 
+      def remove_completed_deploy_pods
+        pods = object_manager.get_pods.select { |e| e.status.phase == "Succeeded" && e.metadata.name.match?(/^collector.*\d+-deploy$/) }
```

Comment:
This will all go away once we teach orchestrator how to deal in `Deployments`?

Reply:
Yep, this will get nuked from orbit when we change from `DeploymentConfigs` -> `Deployments`.